portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,16 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['If I was stuck on a desert island, I would bring a good book', 'My favorite animal is a lizard', 'I grew up in Boston, MA', 'The food I eat most is Cheerios', 'When I graduate, I plan on moving to Australia', 'I learned to surf when I was eight', 'I was the cross country captain for my high school team', 'My favorite color is yellow', 'I am super interested in the ocean and marine life'];
```

Comment:
recommendation is usually to [keep column limit to 80 chars](https://google.github.io/styleguide/jsguide.html#formatting-column-limit).  Going over wouldn't break the functionality, but it's more for cleanliness/readability.